### PR TITLE
added a loader in project submit

### PR DESCRIPTION
**Pull request by kumaryogesh17**
### Fixes Issue 
 #192 

### Changes proposed

Added a loader animation to the submit project page

### Screenshots
Screenshot from 2022-10-01 12-59-47

### CODE REVIEW

**Comment by rohitdasu** [comment]
@leewan-09 did you able to see the loader?

**Review by leewan-09 — CHANGES_REQUESTED** [review]
Lottie loader can't load because it is much larger. I think we should use a lightweight loader.
https://flowbite.com/docs/components/spinner/

image

**Comment by leewan-09** [comment]
> @leewan-09 did you able to see the loader?

nop

File: pages/projects/submit.tsx [hunk 1]
```diff
@@ -45,7 +45,8 @@ const SubmitProject = () => {
   if (status === 'loading') {
     return (
       <div className="h-screen flex items-center justify-center">
-        Loading...
+        {/* Loading... */}
+        <iframe src="https://embed.lottiefiles.com/animation/98194"></iframe>
```
Comment (rohitdasu) [inline pages/projects/submit.tsx:49]: could you use lottie package which exists in our current project to display the animation?
we have implemented for the homepage, see it for reference.

**Review by rohitdasu — CHANGES_REQUESTED** [review]

File: pages/projects/submit.tsx (hunk 1 above)
Comment (kumaryogesh17) [reply pages/projects/submit.tsx]: > could you use lottie package which exists in our current project to display the animation? we have implemented for the homepage, see it for reference.

The triangular loader which is present at right bottom of page?

**Review by leewan-09 — COMMENTED** [review]
Can you fix the build? Build it locally and test whether it is okay.

**Comment by kumaryogesh17** [comment]
> Can you fix the build? Build it locally and test whether it is okay.

okay

File: pages/projects/submit.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ import { useForm, SubmitHandler, Controller } from 'react-hook-form';
 import { useSession } from 'next-auth/react';
 import Router from 'next/router';
 import { motion } from 'framer-motion';
+import Lottie from "lottie-react";
```
Comment (rohitdasu) [inline pages/projects/submit.tsx:10]:
```suggestion
import Lottie from 'lottie-react';
```

File: pages/projects/submit.tsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import { useForm, SubmitHandler, Controller } from 'react-hook-form';
...
+import Loader from "./Loader/98288-loading.json";
```
Comment (rohitdasu) [inline pages/projects/submit.tsx:11]: can you put this json file inside public folder. and please rename the json file to loading.json

**Review by rohitdasu — CHANGES_REQUESTED** [review]

File: pages/projects/submit.tsx (hunk 3 above)
Comment (kumaryogesh17) [reply pages/projects/submit.tsx:11]: yes I will change.

**Comment by rohitdasu** [comment]
@kumaryogesh17 there are prettier build issue. could you fix it

**Review by leewan-09 — CHANGES_REQUESTED** [review]
this is huge and the color should match the theme.

image

**Comment by kumaryogesh17** [comment]
which color should i use ?

**Comment by kumaryogesh17** [comment]
is orange color is fine ?

**Comment by kumaryogesh17** [comment]
> ****

please review I have updated the color of loader.

**Review by rohitdasu — APPROVED** [review]
LGTM 👍

**Pull request merged**